scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -33,13 +33,92 @@ async function main() {
   const sdl = (await ethers.getContract("SDL")) as SDL
   const veSDL = (await ethers.getContract("VotingEscrow")) as VotingEscrow
   const minter = (await ethers.getContract("Minter")) as Minter
-  const DAY = 86400
   const WEEK = 86400 * 7
   const YEAR = WEEK * 52
-  const GaugeHelperContract = (await ethers.getContract(
+  const MAXTIME = 86400 * 365 * 4
```

Comment:
year * 4?

Reply:
+1 on this

Reply:
un-used so removing, but yes thats 4 years in seconds